reference/2_constructing_data/2_4_syncer_splitter/README.md

```diff
@@ -0,0 +1,194 @@
+# Using the flow/Syncer & Splitter nodes for data manipulation
+
+The ```flow/Syncer``` node in Cranq is a synchronization primitive, that combines the data content of it's input signals into a single output stucture. It can be construct both arrays & key-value pairs. 
+
+The syncer node features a dynamic input port set (spread port):
+- It's ```fields``` input takes an array, which determines the quantity and names of its input ports
+- The names of these input ports will determine whether the node yields an array, or a key-value pair collection

```

Comment:
Maybe name "key-value pair collection" a "dictionary"?
Like this "or a dictionary (key-value pair collection)" 
This way we can use the simpler dictionary term further on.

Reply:
Will fix